app/_src/gateway/upgrade/backup-and-restore.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Backup and restore
+content_type: how-to
+purpose: Learn how to back up and restore Kong Gateway
+---
+
+Before you start any upgrade, back up your {{site.base_gateway}} data. 
+Kong supports two back up methods for {{site.base_gateway}} entities: database backup and declarative backup.
+
+We recommend backing up data using both methods, as this offers recovery flexibility.
+* The native database tools are robust and can restore data instantly, compared to the declarative tool. 
+* In case of data corruption, try to do a database-level restore first, otherwise bootstrap a new database and use decK to load in entity data.
```

Comment:
Apart from decK, we have `kong config` command as well. Seems the `kong config` CLI part is removed?

Reply:
Revisiting this. Previous guidance from Rob (on many tickets that I won't track down now) was that we should not document `kong config` as a viable option _at all_, but based on Rob's response [here](https://docs.google.com/document/d/1ddblT7Nb34LfjGLSoIxKQuN0Q87TKAr8dZffoaID2zw/edit?disco=AAAA3sXNSIY), sounds like we can consider it as an okay secondary option.